hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -267,12 +303,15 @@ func (r *HostedClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 
 	ready := currentCluster.Status.ControlPlaneReady && currentCluster.Status.InfrastructureReady
 	if !ready {
-		r.Log.Info("Not ready yet. Requeueing")
-		return ctrl.Result{Requeue: true}, nil
+		r.Log.Info("Not ready yet.")
+		return ctrl.Result{}, nil
```

Comment:
why are we dropping requeing here?

Reply:
because we now rely on watches on child components to respond to changes in them, as opposed to requeueing here, which was effectively polling for changes.